docs/pages/why-web-components.md

```diff
@@ -0,0 +1,124 @@
+
+### Web components help ensure longevity of code
+
+Preferred web stacks evolve. Many mature organizations maintain applications written in a variety of frameworks. Each rewrite or launch is an opportunity to update codebases to the &ldquo;modern approach,&rdquo; but that changes over time.
+
+Web components allow us to take our frontend widgets&nbsp;off this cycle&nbsp;of getting rebuilt in the newest framework flavor. Standardized features are built into all browsers. This means they&rsquo;re supported forever. We are not subject to breaking changes in new versions of frameworks. We also do not worry about developers who prefer to write widgets that only work with a specific underlying technology.
+
+### Easier component management
+
+Publishing each component individually lets us take advantage of separate versioning. Site owners get <a href="https://github.com/dependabot">dependabot</a>&nbsp;notifications when there are updates to each npm package. This comes with associated <a href="https://semver.org/">semver</a> compliant version numbers and published changelogs. Site owners can decide if they want to upgrade or ignore the change on their own timeline.
+
+Owners can add, upgrade, or remove components 
+
+more easily than if its code was part of the whole site. We learn a lot from other modular design systems like:
+
+- The <a href="https://designsystem.digital.gov/">U.S. Web Design System</a>
+- Other systems also built with web components, like <a href="https://opensource.adobe.com/spectrum-web-components/">Adobe&rsquo;s Spectrum</a>
+- Systems with fantastic documentation for new contributors, like&nbsp;<a href="https://opensource.adobe.com/spectrum-web-components/">Clarity</a>
+
+### Components encapsulate code
+
+The code for each widget is within the custom element class. Our component patterns with local variables protect us from accidentally influencing global variables. The custom element API provides a clear interface for interacting with the rest of the application. It receives events responding to instantiation, removal, and attribute updates.
+
+## When we use web components
+
+We only need web components when we need JavaScript. We do not always need JavaScript. If HTML and CSS provide the desired features, that&rsquo;s all a component uses.
+
+## Progressive enhancement
+
+We cannot depend&nbsp;on every device supporting our web components, even if they work in the latest version of all browsers. Older devices get locked out of update cycles. Some people avoid freely available software updates. 
+
+Critical components should work even if the JavaScript fails. For example, accordions have always been progressively enhanced. Now they use details and summary elements. This lets people read the content, even if the JavaScript fails to execute. This failure state eliminates the extra styling or open/close animations.
```

Comment:
"accordions have always been progressively enhanced" Is that a reference to our accordions? Do we need that sentence?

Suggest something like: 
"We recently refactored our accordions. Now they use.."

Reply:
Sounds good to me. What do you think @ashleydraper?

Reply:
Change to `<details>` and `<summary>` elements. I think the tags make it more obvious what you're referring to.

Reply:
updated